### PR TITLE
Enchancement(fuse-subgraph): reduce subgraph size

### PR DESCRIPTION
**Pull request by mul53**
- Remove templates

### CODE REVIEW

File: fuse-subgraph/subgraph.template.yaml [hunk 1]
```diff
@@ -138,49 +138,6 @@ dataSources:
           handler: handleTokenRegistered
       file: ./src/mappings/entities.ts
 templates:
```
Comment (leonprou) [inline fuse-subgraph/subgraph.template.yaml:140]: you can also delete `templates` I think

**Review by leonprou — APPROVED** [review]
Looks good 👍

File: fuse-subgraph/subgraph.template.yaml (hunk 1 above)
Comment (mul53) [reply fuse-subgraph/subgraph.template.yaml:140]: https://github.com/fuseio/fuse-graph/blob/d129edebf50073334e6c31f3bad0052ebce1b04d/fuse-subgraph/src/mappings/entities.ts#L26 we have this line which depends on this

Comment (leonprou) [reply fuse-subgraph/subgraph.template.yaml:140]: got ya

**Pull request merged**